### PR TITLE
Lesson 6 Task 9 - Divisibility by 3

### PR DESCRIPTION
**Pull request by jean-farinea**
1. Class created to check if number is divisible by three within exception custom message

### CODE REVIEW

File: src/main/java/lesson6/task9/DivisibilityByThree.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package lesson6.task9;
+
+public class DivisibilityByThree {
```
Comment (strzeleckik) [inline src/main/java/lesson6/task9/DivisibilityByThree.java]: different name: DisibilityChecker

File: src/main/java/lesson6/task9/DivisibilityByThree.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    }
+
+    private static void checkDivisibility(int number) {
+        if (number % 3 != 0) {
+            throw new IllegalArgumentException(number + " is not divisible by 3.");
+        }
+    }
+
+    public static void main(String[] args) {
```
Comment (strzeleckik) [inline src/main/java/lesson6/task9/DivisibilityByThree.java]: unit test

**Review by strzeleckik — CHANGES_REQUESTED** [review]
small changes

**Comment by jean-farinea** [comment]
Changes done, please review.

File: src/test/java/lesson6/task9/DivisibilityCheckerTest.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package lesson6.task9;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DivisibilityCheckerTest {
+
+    @Test
+    void shouldCheckDivisibilityByThree() {
```
Comment (strzeleckik) [inline src/test/java/lesson6/task9/DivisibilityCheckerTest.java:10]: you are testing positive and negative scenario in one test, thats a bad practice

**Comment by jean-farinea** [comment]
Test adjusted, please review.

**Review by strzeleckik — APPROVED** [review]
ok

**Pull request merged**